src/interface.jl

```diff
@@ -9,23 +9,25 @@ const IBF = IntervalBasedForest
 end
 
 function MMI.fit(m::RandomForestClassifierTS, verbosity::Int, X, y)
-    X, y = MMI.matrix(X), array(y)
+    X, yplain = MMI.matrix(X), MMI.int(y)
+    classes_seen  = filter(in(unique(y)), MMI.classes(y[1]))
+    integers_seen = MMI.int(classes_seen)
     tree = IBF.randomforestflassifierFit(X, y,
                                     n_trees=m.n_trees,
                                     min_interval=m.min_interval,
                                     pruning_purity_threshold=m.pruning_purity_threshold)
-    fitresult = tree
+    fitresult = (tree, classes_seen, integers_seen)
     cache  = nothing
 
     return fitresult, nothing, nothing
```

Comment:
Should that be `return fitresult, cache, report` or so?

Reply:
Return statement does the same, in our case we don't need `cache, report` yet. & yes `fitresult = (tree, classes_seen, integers_seen)` is correct.